assembly/std/storage.ts

```diff
@@ -188,49 +287,71 @@ export function appendOf<T>(address: Address, key: T, value: T): void {
 }
 
 /**
- * Checks if the (key, value) exists in the datastore
- * of the callee's address.
+ * Checks if the key-value pair exists in the datastore of the current contract.
+ *
+ * @typeParam T - Can be either `string`, `Args`, or `StaticArray<u8>`.
+ *
+ * @param key - The key to check for existence in the datastore.
  *
- * @typeParam T - `string`, `Args` or `StaticArray<u8>`
- * @param key -
+ * @returns A boolean value indicating whether the key-value pair exists in the datastore or not.
+ *
+ * @throws
+ * - [compilation only] if the `key` type are neither `string`, `StaticArray<u8>`, or `Args`.
  */
 export function has<T>(key: T): bool {
   return env.has(toDatastoreFormat(key));
 }
 
 /**
- * Checks if the (key, value) exists in the datastore
- * of the given address.
+ * Checks if the key-value pair exists in the datastore of the target address.
+ *
+ * @remarks Can only be called at smart contract generation time by the parent smart contract to the child's address.
+ * 
+ * @typeParam T - Can be either `string`, `Args`, or `StaticArray<u8>`.
+ * 
+ * @param address - The child smart contract `address`.
+ * @param key - The key to check for existence in the datastore.
  *
- * @typeParam T - `string`, `Args` or `StaticArray<u8>`
- * @param address -
- * @param key -
+ * @returns A boolean value indicating whether the key-value pair exists in the datastore or not.
  *
+ * @throws
+ * - if the contract at the given `address` does not exist
+ * - if the caller is not authorized to modify the datastore.
+ * - [compilation only] if the `key` type are neither `string`, `StaticArray<u8>`, or `Args`.
  */
 export function hasOf<T>(address: Address, key: T): bool {
   return env.hasOf(address.toString(), toDatastoreFormat(key));
 }
 
 /**
- *  Sets the executable bytecode of the callee's address.
+ * Sets the executable bytecode of the current contract.
```

Comment:
What is the bytecode and what would we need to set it ?

Reply:
Bytecode is the raw representation of a WebAssembly file. To set it properly you need to have:
* a smart contract using the right AS std (which maps the massa node v.x.x execution VM ABIs state)
* compile this sc into a WASM file
* use the byte array representation of that file

Reply:
I'm not sure that answers your question let me know if it's not the case

Reply:
Thanks for the answer, do you have a use case where we would set the bytecode for the curent address ?

Reply:
From within an SC that would want to update itself for `setBytecode`. Or in a deployement SC to update the bytecode of an address that was created in the same scope (case where another address has ownership over it) for `setBytecodeOf`